### PR TITLE
Dilithium Crystal Tweaks

### PR DESCRIPTION
**Pull request by nmajask**
# General Documentation

### Intent of your Pull Request

Changes a lot with Dilithium Crystals. In short Dilithium Crystals now works as a mat, like metal or bluespace crystals. Higher tiers of batteries now require Dilithium Crystals, though this is more of a proof of concept and is probably not very balanced. Suggestions on how it should be used and feedback on how it is currently being used would be nice. Also, Dilithium Crystals are missing a crush effect still, so if anyone has a good idea for that, please comment it.

### Why is this change good for the game?

It makes Dilithium Crystals fit in more with the rest of the ores instead of just getting eaten by the ORM. Also allows engineers to get Dilithium Crystals directly from their techfab like they would metal or glass.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Dilithium Crystals now works as a mat, like metal or bluespace crystals, and are used for some power cells.
### What should players be aware of when it comes to the changes your PR is implementing?
The new cost of some cells and miners now need to get Dilithium Crystals
### What general grouping does this PR fall under? 
Uhhh... Mining Tweaks?
### Are there any aspects of the PR that you would like us not to mention on the Wiki?
Nope
### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
Dilithium Crystals are worth 50 points
Super-Capacity Power Cells now requires 250 Dilithium Crystal
Hyper-Capacity Power Cells and Bluespace Power Cells now require 500 Dilithium Crystal
# Changelog
:cl:  
tweak: Made Dilithium crystal a material 
tweak: Made some higher tiers of power cells require Dilithium Crystal
rscadd: Added Dilithium Polycrystal and Refined Dilithium
/:cl:

### CODE REVIEW

**Review by nesisknesis — APPROVED** [review]

**Comment by JamieD1** [comment]
Can you check your line endings, they seem a bit wonky.

**Comment by nmajask** [comment]
Sorry, could you give some examples, I don't know what you mean.

**Comment by JamieD1** [comment]
> Sorry, could you give some examples, I don't know what you mean.

Can you make sure your line endings are the same as the file that you are editing, most likely CRLF

**Comment by nmajask** [comment]
Did the last commit fix it?

**Comment by JamieD1** [comment]
> Did the last commit fix it?

Basemats still has the issue, otherwise its fine.

Can you check it again for me?

**Comment by nmajask** [comment]
> Basemats still has the issue, otherwise its fine.
> 
> Can you check it again for me?

Oh, are you talking about the whitespace changes? I think those are caused by how I uploaded it. If that was the issue, it should be fixed now.

**Comment by nmajask** [comment]
I'm going to be busy till the 6th and I will be unable to fix the inevitable issues/complaints, so if you want to hold off merging or whatever your doing with this until then.

**Comment by nmajask** [comment]
One day this will get merged
